tests/utils.rs

```diff
@@ -70,3 +76,19 @@ pub fn get_toml(manifest_path: &str) -> toml_edit::Document {
     f.read_to_string(&mut s).unwrap();
     s.parse().expect("toml parse error")
 }
+
+pub fn get_command_path(s: impl AsRef<OsStr>) -> String {
+    let target_dir: PathBuf = env::var_os("CARGO_TARGET_DIR")
+        .unwrap_or_else(|| OsString::from("target"))
+        .into();
+
+    let mut binary_name = OsString::from("cargo-");
+    binary_name.push(s.as_ref());
+
+    target_dir
+        .join("debug")
```

Comment:
Does it work it `--release` mode? Do we want to check for `PROFILE`?

Reply:
Oh, good point. Do we ever run the tests in release mode right now?

Reply:
We don't, but someone might, e.g. we in the future :P

Reply:
Is it out of scope of the PR, i.e. do you want to address it in a separate PR?